packages/passport/sdk/src/errors/passportError.ts

```diff
@@ -52,6 +54,9 @@ export const withPassportError = async <T>(
       errorMessage = (error as Error).message;
     }
 
-    throw new PassportError(errorMessage, customErrorType);
+    const passportError = new PassportError(errorMessage, customErrorType);
+    trackError('passport', customEventName || '', passportError);
```

Comment:
It looks like there are 15 places that call `withPassportError` without specifying a `customEventName`. Can we either:
- make `customEventName` mandatory, OR
- only call `trackError` if `customEventName` is set

That way we won't end up with a whole barrage of `passport.trackError_` logs.

Reply:
Have logged [ID-2222](https://immutable.atlassian.net/browse/ID-2222) for the Passport team to action at a later date.